doc/sphinx/proofs/writing-proofs/equality.rst

```diff
@@ -508,8 +508,8 @@ which reduction engine to use.  See :ref:`type-cast`.)  For example:
    then only the named reductions are applied.  The reductions include:
 
    `head`
-     Do only head reduction, without going under binders.  Only
-     supported by :tacn:`simpl`, :tacn:`cbn` and :tacn:`lazy`.
+     Do only head reduction, without going under binders.
+     Supported by :tacn:`simpl`, :tacn:`cbv`, :tacn:`cbn` and :tacn:`lazy`.
      If this is the only specified reduction, all other reductions are applied.
```

Comment:
These items should appear in the same order they are given in the grammar (lines 496-503.  Please reorder the grammar items (in orderedGrammar) or the position of this text.

Reply:
OK, will do.